api_pagopa.yaml

```diff
@@ -200,6 +202,23 @@ definitions:
         maxLength: 36
     required:
     - idPagamento
+  SoggettoPagatore:
+    type: object
+    properties:
+      tipo:
+        type: string
+        minLength: 1
+        maxLength: 2
+      fiscal_code:
+        $ref: "#/definitions/FiscalCode"
+      anagrafica:
+        type: string
+        minLength: 1
+        maxLength: 71
```

Comment:
check whether these limits are really enforced by the pagopa-nodo, perhaps it would be better to be more permissive on our side and just accept non empty strings

Reply:
The type definition follows the specs defined in [here](https://github.com/pagopa/io-pagopa-api/blob/master/wsdl/nodo/PagInf_RPT_RT_6_2_0.xsd#L337). Other types use a strict definition too. Can you link the repo that contains the nodo attiva implementation?